includes/discovery/storage/eql-storage.inc.php

```diff
@@ -19,7 +19,7 @@
     foreach ($eql_storage as $index => $storage) {
         $fstype = $storage['eqliscsiVolumeAdminStatus'];
         $descr  = $storage['eqliscsiVolumeName'];
-        $units  = 1;
+        $units  = 1000000;
```

Comment:
I'm always worried about changing things like that in that it can vary from firmware to firmware. Have you confirmed this actually needs changing full stop?